tools/install_opte.sh

```diff
@@ -142,18 +142,34 @@ ensure_helios_dev_is_non_sticky
 add_publisher "$HELIOS_NETDEV_REPO_PATH"
 add_publisher "$XDE_REPO_PATH"
 
-# Actually update packages, handling case where no updates are needed
+# Actually install the xde kernel module and opteadm tool
 RC=0
-pkg update || RC=$?;
```

Comment:
Is the explicit update not needed if you use `pkg://helios-netdev`?

Reply:
I used `pkg update` to do the update of the `on-nightly` publisher, which is what provides the new kernel bits. The issue is that, the `helios-dev` (stock Helios) is now "beyond" that in terms of its version number. That means that those bits won't be installed via `pkg update`.

I'm trying to be more explicit in this commit in general. I specify exactly which publisher and package to install, both when we're installing OPTE and uninstall it.